src/Phansible/Roles/Xdebug.php

```diff
@@ -34,4 +36,17 @@ public function getInitialValues()
           'settings' => [],
         ];
     }
+
+    public function transformValues(array $values, VagrantBundle $vagrantBundle)
+    {
+        if ($values['install'] == 1) {
+            $values['settings'] = [
```

Comment:
Either way I can try to use some hidden fields and `getInitialValues ` if you prefer that for consistency reasons.